src/app/static/src/tests/components/runWorkflow/workflowSummary/workflowSummaryReports.test.ts

```diff
@@ -145,4 +174,21 @@ describe(`workflowSummaryReports`, () => {
 
     });
 
+    it(`it can render depends on dependencies and missing dependencies`, () => {
```

Comment:
Could add a little more detail to this test e.g. to check it's not trying to render dependencies for the report which doesn't have any, and that it's rendering dependencies and missing dependencies for the correct reports e.g. could check expected elements are found in the correct index of `div.single-workflow-summary-area`.